### ISSUE TITLE
Display tags on website

### ISSUE THREAD

**Issue by noraj** [dendronhq/dendron#604]
## Please Select if your Request is Either something new or an Enhancement ##

- [x] Enhancement of an existing Feature.
- [ ] Request of a new feature.

## Please select the area your request applys to. (Multiple selections are Possible) ##

- [x] UI
- [ ] Workbench
- [x] Publishing
- [ ] Pods
- [ ] Markdown
- [ ] Hierarchy
- [ ] Graph
- [ ] Misc (Choose this if your not sure)

## Is your feature request related to a problem? Please describe ##

No

## Describe the solution you'd like ##

When [publishing to website](https://wiki.dendron.so/notes/230d0ccf-5758-4a8f-b39b-3b68e1482e2b.html), it would be nice to have the list of tags of the page at the end of the page, like on Docker or Amazon documentation.

## Describe alternatives you've considered ##

It could also be at the top of the page under the breadcrumb.

## Additional context ##

![image](https://user-images.githubusercontent.com/16578570/113515362-5cc39a80-9574-11eb-9eec-f7cff6104e6f.png)

**Comment by flammehawk** [comment]
For that, I think we would need a way to specify what a tag actually is. 
Currently, they are Hierarchies as well. 
Possible solutions to that would be to extend schemas to flag a hierarchy as a tag collection.

**Comment by funnym0nk3y** [comment]
Yeah, I'd like to see that too. How about defining the tags outside of the actual note tree maybe in a seperate tag "environment"? Navigating tags in the hierarchy is a little cumbersome.
